webplatformcompat/view_serializers.py

```diff
@@ -21,7 +21,14 @@
 from .serializers import (
     BrowserSerializer, FieldMapMixin, FeatureSerializer, MaturitySerializer,
     SectionSerializer, SpecificationSerializer, SupportSerializer,
-    VersionSerializer, omit_some)
+    VersionSerializer, FieldsExtraMixin)
+
+
+def omit_some(source_list, *omitted):
+    """Return a list with some items omitted"""
+    for item in omitted:
+        assert item in source_list, '%r not in %r' % (item, source_list)
```

Comment:
Also, is `'%r not in %r'`"unicode-safe"? i.e. if one of those things has a `__repr__` that returns a unicode with non-ascii characters, does the message for the failed assertion kick up its own UnicodeEncodeError?

I don't think `__repr__` should be returning unicodes, but life is an adventure sometimes.